binderhub/static/js/index.js

```diff
@@ -92,16 +82,14 @@ function build(providerSpec, log, fitAddon, path, pathType) {
     image.close();
   });
 
-  image.onStateChange('built', function(oldState) {
-    if (oldState === null) {
-      $('#phase-already-built').removeClass('hidden');
-      $('#phase-launching').removeClass('hidden');
-    }
+  image.onStateChange('built', function() {
+    $('#phase-already-built').removeClass('hidden');
+    $('#phase-launching').removeClass('hidden');
     $('#phase-launching').removeClass('hidden');
```

Comment:
This line is a duplicate

Reply:
Thanks for revieiwng @manics! I opened https://github.com/jupyterhub/binderhub/pull/1772 as I was too quick on the ball